<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Setup</class>
 <widget class="QDialog" name="Dialog_Setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../obpm.qrc">
    <normaloff>:/png/png/setup.png</normaloff>:/png/png/setup.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Alias</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit_user1">
        <property name="toolTip">
         <string>enter name for user 1</string>
        </property>
        <property name="text">
         <string>User 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_user2">
        <property name="toolTip">
         <string>enter name for user 2</string>
        </property>
        <property name="text">
         <string>User 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_1">
     <property name="title">
      <string>Defaults</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="3">
         <widget class="QLabel" name="label_sys">
          <property name="text">
           <string notr="true">135</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_bpm">
          <property name="text">
           <string notr="true">80</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../obpm.qrc">:/png/png/bpm.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSlider" name="horizontalSlider_sys">
          <property name="minimum">
           <number>115</number>
          </property>
          <property name="maximum">
           <number>155</number>
          </property>
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>135</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSlider" name="horizontalSlider_bpm">
          <property name="minimum">
           <number>60</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>80</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../obpm.qrc">:/png/png/dia.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSlider" name="horizontalSlider_dia">
          <property name="minimum">
           <number>65</number>
          </property>
          <property name="maximum">
           <number>105</number>
          </property>
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>85</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_dia">
          <property name="text">
           <string notr="true">85</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_1">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../obpm.qrc">:/png/png/sys.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string notr="true">SYS</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string notr="true">DIA</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string notr="true">BPM</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Diagram</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox_legend">
        <property name="text">
         <string>show Legend with min/Ã˜/max Values</string>
        </property>
        <property name="icon">
         <iconset resource="../obpm.qrc">
          <normaloff>:/png/png/legend.png</normaloff>:/png/png/legend.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_point">
          <property name="text">
           <string>Point</string>
          </property>
          <property name="icon">
           <iconset resource="../obpm.qrc">
            <normaloff>:/png/png/chart-point.png</normaloff>:/png/png/chart-point.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_line">
          <property name="text">
           <string>Line</string>
          </property>
          <property name="icon">
           <iconset resource="../obpm.qrc">
            <normaloff>:/png/png/chart-line.png</normaloff>:/png/png/chart-line.png</iconset>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_step">
          <property name="text">
           <string>Step</string>
          </property>
          <property name="icon">
           <iconset resource="../obpm.qrc">
            <normaloff>:/png/png/chart-step.png</normaloff>:/png/png/chart-step.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_impulse">
          <property name="text">
           <string>Impulse</string>
          </property>
          <property name="icon">
           <iconset resource="../obpm.qrc">
            <normaloff>:/png/png/chart-impulse.png</normaloff>:/png/png/chart-impulse.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Update</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkBox_update">
        <property name="text">
         <string>check for Online Updates when Program starts</string>
        </property>
        <property name="icon">
         <iconset resource="../obpm.qrc">
          <normaloff>:/png/png/work.png</normaloff>:/png/png/work.png</iconset>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_reset">
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="icon">
        <iconset resource="../obpm.qrc">
         <normaloff>:/png/png/reset.png</normaloff>:/png/png/reset.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../obpm.qrc">
         <normaloff>:/png/png/save.png</normaloff>:/png/png/save.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../obpm.qrc"/>
 </resources>
 <connections/>
</ui>
